client/blocks/stats-navigation/index.js

```diff
@@ -167,6 +168,11 @@ class StatsNavigation extends Component {
 
 		// @TODO: Add loading status of modules settings to avoid toggling modules before they are loaded.
 
+		// The isNewSite value can be null so we need to guard against that.
+		// This is semantically incorrect as we expect a boolean value.
+		// TODO: Update logic in connect() function to send only true/false value.
+		const delayTooltipPresentation = isNewSite === null || isNewSite;
```

Comment:
Could we use a more specific naming to describe the relation with the `isNewSite,` e.g., `showTooltipForOldSite`, to possibly reduce the annotations?
```suggestion
		const showTooltipForOldSite = isNewSite === false;
```

Reply:
The extra comments shouldn't be needed once I fix up the logic in the `connect()` function. I'll look to update the property name to better represent the updated logic too.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/95509.